### ISSUE TITLE
Disco windows

### ISSUE THREAD

**Issue by jboyens** [ingolemo/firefox-dynamic-zoom#7]
When you have two windows open to the same domain, the plugin fights with itself over which zoom is correct. It creates this wonderful disco flicker effect that is fun to watch, but difficult to cope with. 

Love the extension, thank you so much.

**Comment by ingolemo** [comment]
Yeah, unfortunately firefox doesn't allow web extensions to use per-tab zooming, despite having the API for it. This issue can be worked around by going into `about:config` and setting `browser.zoom.siteSpecific` to `false`.

https://bugzilla.mozilla.org/show_bug.cgi?id=1319387
